libs/ledger-lib-verification/src/main/kotlin/net/corda/ledger/libs/verification/impl/UtxoLedgerUtxoTransactionVerifierImpl.kt

```diff
@@ -0,0 +1,46 @@
+package net.corda.ledger.libs.verification.impl
+
+import net.corda.ledger.libs.verification.UtxoTransactionVerifier
+import net.corda.ledger.libs.verification.impl.VerificationUtils.verifyCommands
+import net.corda.ledger.libs.verification.impl.VerificationUtils.verifyInputsAndOutputs
+import net.corda.ledger.libs.verification.impl.VerificationUtils.verifyNoDuplicateInputsOrReferences
+import net.corda.ledger.libs.verification.impl.VerificationUtils.verifyNoInputAndReferenceOverlap
+import net.corda.ledger.libs.verification.impl.VerificationUtils.verifySignatories
+import net.corda.v5.ledger.utxo.transaction.UtxoLedgerTransaction
+
+class UtxoLedgerUtxoTransactionVerifierImpl(
```

Comment:
```suggestion
class UtxoLedgerTransactionVerifierImpl(
```

I don't like the double UTXO in the name.